WikipediaLibrary/TWLight
# Pull partner descriptions from JSON files

## Description
Get the partner descriptions from a JSON file instead of the database and display them on the Django templates.

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
Partner descriptions could not be translated to a user's language of choice. This change helps solve this.

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T278337](https://phabricator.wikimedia.org/T278337)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)
1. Create a `partner_descriptions.json` file in a language other than English
2. Add a key-pair combination of your partner of choice
3. Change your settings to that language
4. Ensure that the partner description is in the chosen language

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)
**Partner description when the description exists in the user's language of choice**
![Screen Shot 2021-04-05 at 17 35 47](https://user-images.githubusercontent.com/7854953/113640950-34a26d00-9642-11eb-9a62-6c528e373532.png)


**Partner description when the description does not exist in the user's language of choice**
![Screen Shot 2021-04-05 at 17 36 00](https://user-images.githubusercontent.com/7854953/113640971-3f5d0200-9642-11eb-97cc-f1db9fbb16b6.png)


## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
